src/Bridge/NelmioApiDoc/Extractor/ApiDocExtractor.php

```diff
@@ -0,0 +1,46 @@
+<?php
```

Comment:
This is no longer needed since `nelmio/api-doc-bundle` `^2.10`. I will bump the version requirement and remove this.